translate/storage/aresource.py

```diff
@@ -178,6 +178,9 @@ def __init__(self, source, empty=False, xmlelement=None, **kwargs):
     def istranslatable(self):
         return bool(self.getid()) and self.xmlelement.get("translatable") != "false"
 
+    def marktranslateable(self, value):
+        return self.xmlelement.set("translatable", value)
```

Comment:
As the getter is called `istranslatble`, this should be named consistently. Also I'd prefer it to accept boolean in the API as the string values are Android-specific:


```suggestion
    def marktranslatable(self, value: bool) -> None:
        return self.xmlelement.set("translatable", "true" if value else "false")
```

Reply:
Updated the code based on your suggestion. I agree on the point of making `marktranslatable` function to accept boolean value. This will make its implementation to other format easier.